sky/backends/backend_utils.py

```diff
@@ -903,6 +903,8 @@ def write_cluster_config(
                 'ray_temp_dir': constants.SKY_REMOTE_RAY_TEMPDIR,
                 'dump_port_command': dump_port_command,
                 # Ray version.
+                'RAY_CMD': constants.SKY_RAY_CMD,
+                'SKY_PIP_CMD': constants.SKY_PIP_CMD,
```

Comment:
Bit inconsistent -- w/ L908. Use lower_case? L905 -> Sky constants.

Reply:
Changed to lower case, thanks!